runtime/drivers/bigquery/warehouse.go

```diff
@@ -263,6 +264,7 @@ func (f *fileIterator) Next() ([]string, error) {
 			if err := writer.WriteBuffered(rec); err != nil {
 				return nil, err
 			}
+			rows++
```

Comment:
One arrow record can have multiple rows as well.

Reply:
ok so we can use `rec.NumRows()` to add to `rows`, should we do this or just use `TotalRecords` field on `FileIterator` ?

Reply:
I think lets calculate rows ourselves so that if we miss something due to any bug we can see that

Reply:
Sure sounds good.
Lets log the number of rows from Bigquery(TotalRecords) as well as based on row group records(rec.NumRows()) since there could be a bug in parquet generation as well. 
This will rule out near impossible case that `BigQuery` itself is returning less rows.

Reply:
We can also make this and the other log `Info` since I don't think we capture debug logs.